spectrocloud/resource_cluster_aws.go

```diff
@@ -267,6 +271,22 @@ func resourceClusterAws() *schema.Resource {
 								Type: schema.TypeString,
 							},
 						},
+						"az_subnet": {
```

Comment:
Let's consider here similar to EKS schema or refactor EKS to use new schema.

Reply:
We cannot use same as EKS, In EKS schema we cant have two subnet with same availability zones, as below examples, @nikchern we need to think about fixing eks schema as well

Eg: 

```
az_subnets = {
      "us-west-2a" = "subnet-0d4978ddbff16c"
      "us-west-2a" = "subnet-0d4978ddbff16c"
    }
```
or
```
az_subnets = {
      "us-west-2a" = "subnet-0d4978ddbff16c"
    }
    az_subnets = {
      "us-west-2a" = "subnet-0d4978ddbff16c"
    }
```

Reply:
@kreeuwijk as discussed with @karl-cardenas-coding we will go with a more verbose schema: 

```
 az_subnet {
      subnet_id = "subnet-0fd3677d9c41c2d82"
      az_id = "us-west-2a"
    }
```

Reply:
With this approach, we can use a `dynamic` block 😄